client/homescreen/activity-panel/index.js

```diff
@@ -39,16 +39,14 @@ export const ActivityPanel = () => {
 		const countLowStockProducts = getLowStockCount( select );
 		const countUnapprovedReviews = getUnapprovedReviews( select );
 		const publishedProductCount = getSetting( 'publishedProductCount', 0 );
-		const taskLists = select( ONBOARDING_STORE_NAME ).getTaskLists();
+		const taskList = select( ONBOARDING_STORE_NAME ).getTaskList( 'setup' );
 
 		return {
 			countLowStockProducts,
 			countUnapprovedReviews,
 			countUnreadOrders,
 			manageStock,
-			isTaskListHidden: Boolean( taskLists.length )
-				? ! taskLists.find( ( list ) => list.id === 'setup' ).isVisible
-				: null,
+			isTaskListHidden: taskList?.isVisible,
```

Comment:
@joshuatf this should be `isHidden` right?

Reply:
Thanks for catching this!  It actually is supposed to be `isVisible` but the references to `isTaskListHidden` needed to be updated.

I think it's better to check visibility in this case since that's really the dependency we're checking and we know the data has been fully loaded if this value is `true`.

Updated in https://github.com/woocommerce/woocommerce-admin/pull/7820/commits/b957d70a1fe223f0ca7290b861d95f6e0b54a890.